test/ir_Mitsubishi_test.cpp

```diff
@@ -1500,3 +1500,230 @@ TEST(TestDecodeMitsubishiAC, Issue891) {
+
+  ac.setMode(kMitsubishi112Dry);
+  EXPECT_EQ(kMitsubishi112Dry, ac.getMode());
+}
+
+TEST(TestMitsubishi112Class, FanSpeed) {
+  IRMitsubishi112 ac(0);
+  ac.begin();
+
+  ac.setFan(kMitsubishi112FanMax);
+  EXPECT_EQ(kMitsubishi112FanMax, ac.getFan());
+
+  ac.setFan(kMitsubishi112FanMin);
+  EXPECT_EQ(kMitsubishi112FanMin, ac.getFan());
+
+  ac.setFan(255);
+  EXPECT_EQ(kMitsubishi112FanMax, ac.getFan());
+
+  ac.setFan(kMitsubishi112FanMed);
+  EXPECT_EQ(kMitsubishi112FanMed, ac.getFan());
+
+  ac.setFan(kMitsubishi112FanLow);
+  EXPECT_EQ(kMitsubishi112FanLow, ac.getFan());
+
+  ac.setFan(kMitsubishi112FanQuiet);
+  EXPECT_EQ(kMitsubishi112FanQuiet, ac.getFan());
+}
+
+
+TEST(TestMitsubishi112Class, SwingV) {
+  IRMitsubishi112 ac(0);
+  ac.begin();
+
+  ac.setSwingV(kMitsubishi112SwingVAuto);
+  EXPECT_EQ(kMitsubishi112SwingVAuto, ac.getSwingV());
+
+  ac.setSwingV(kMitsubishi112SwingVAuto + 1);
+  EXPECT_EQ(kMitsubishi112SwingVAuto, ac.getSwingV());
+
+  ac.setSwingV(kMitsubishi112SwingVLowest);
+  EXPECT_EQ(kMitsubishi112SwingVLowest, ac.getSwingV());
+
+  ac.setSwingV(kMitsubishi112SwingVLow);
+  EXPECT_EQ(kMitsubishi112SwingVLow, ac.getSwingV());
+
+  ac.setSwingV(kMitsubishi112SwingVHighest);
+  EXPECT_EQ(kMitsubishi112SwingVHighest, ac.getSwingV());
+
+  ac.setSwingV(kMitsubishi112SwingVHigh);
+  EXPECT_EQ(kMitsubishi112SwingVHigh, ac.getSwingV());
+}
+
+TEST(TestMitsubishi112Class, toCommon) {
+  IRMitsubishi112 ac(0);
+  ac.setPower(true);
+  ac.setMode(kMitsubishi112Dry);
+  ac.setTemp(22);
+  ac.setFan(kMitsubishi112FanQuiet);
+  ac.setSwingV(kMitsubishi112SwingVAuto);
+  // Now test it.
+  ASSERT_EQ(decode_type_t::MITSUBISHI112, ac.toCommon().protocol);
+  ASSERT_EQ(-1, ac.toCommon().model);
+  ASSERT_TRUE(ac.toCommon().power);
+  ASSERT_TRUE(ac.toCommon().celsius);
+  ASSERT_EQ(22, ac.toCommon().degrees);
+  ASSERT_EQ(stdAc::opmode_t::kDry, ac.toCommon().mode);
+  ASSERT_EQ(stdAc::fanspeed_t::kMin, ac.toCommon().fanspeed);
+  ASSERT_EQ(stdAc::swingv_t::kAuto, ac.toCommon().swingv);
+  ASSERT_TRUE(ac.toCommon().quiet);
+  // Unsupported.
+  // ASSERT_FALSE(ac.toCommon().swingh);
+  ASSERT_FALSE(ac.toCommon().turbo);
+  ASSERT_FALSE(ac.toCommon().clean);
+  ASSERT_FALSE(ac.toCommon().light);
+  ASSERT_FALSE(ac.toCommon().econo);
+  ASSERT_FALSE(ac.toCommon().filter);
+  ASSERT_FALSE(ac.toCommon().beep);
+  ASSERT_EQ(-1, ac.toCommon().sleep);
+  ASSERT_EQ(-1, ac.toCommon().clock);
+}
+
+TEST(TestMitsubishi112Class, toCommonMode) {
+  ASSERT_EQ(stdAc::opmode_t::kCool,
+            IRMitsubishi112::toCommonMode(kMitsubishi112Cool));
+  ASSERT_EQ(kMitsubishi112Cool,
+            IRMitsubishi112::convertMode(stdAc::opmode_t::kCool));
+  ASSERT_EQ(stdAc::opmode_t::kDry,
+            IRMitsubishi112::toCommonMode(kMitsubishi112Dry));
+  ASSERT_EQ(kMitsubishi112Dry,
+            IRMitsubishi112::convertMode(stdAc::opmode_t::kDry));
+}
+
+
+
+// Decode a 'real' example.
+// Ref: https://github.com/crankyoldgit/IRremoteESP8266/issues/888
```

Comment:
Please remove (or change) this reference. It's for the Mitsubishi136 protocol, not this one.